src/app/leverage/layout.tsx

```diff
@@ -17,7 +17,7 @@ export const metadata = {
 export default function Layout({ children }: LayoutProps) {
   return (
     <Providers>
-      <div className='bg-ic-black dark flex flex-col'>
+      <div className='bg-ic-black dark flex flex-col overflow-x-hidden'>
```

Comment:
this is a fix for the animation bg, as it comes with a bigger width than the background.